stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/TextFieldController.kt

```diff
@@ -152,17 +156,15 @@ class SimpleTextFieldController constructor(
      * This is called when the value changed to is a display value.
      */
     override fun onValueChange(displayFormatted: String): TextFieldState? {
-        val originalTextStateValue = _fieldState.value
```

Comment:
Before: we made two different emissions on an atomic operation.
Now: we emit the two values wrapped in the same data class.